public/js/components/RightSidebar.css

```diff
@@ -68,3 +68,7 @@ span.pause-exceptions.uncaught {
 span.pause-exceptions.all {
   stroke: var(--theme-highlight-blue);
 }
+
+.accordion-button {
+  float: right;
+}
```

Comment:
This works, but it might be nice to try flex as well. An example of that is the autocomplete layout.